.github/workflows/check_doc_build.yml

```diff
@@ -0,0 +1,28 @@
+name: Check documentation build
+
+on:
+  push:
+    branches: [ main ]
+
+  pull_request:
+    branches: [ main ]
+
+jobs:
+  build_and_package:
+    runs-on: ubuntu-latest
+    defaults:
+      run:
+        shell: bash -l {0}
+
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          repository: 'huggingface/optimum-habana'
+          path: optimum-habana
+
+      - name: Make documentation
+        run: |
+          cd optimum-habana
+          git fetch origin
+          git checkout documentation
```

Comment:
This line should be removed before merging - it's included here to test the action works as intended